stdlib/public/core/StringLegacy.swift

```diff
@@ -238,8 +238,8 @@ extension String {
   ///     let max = String(Int.max)
   ///     print("\(max) has \(max.utf16.count) digits.")
   ///     // Prints "9223372036854775807 has 19 digits."
-  public init<T : _SignedInteger>(_ v: T) {
-    self = _int64ToString(v.toIntMax())
+  public init<T : FixedWidthInteger>(_ v: T) {
+    self = _int64ToString(Int64(v))
```

Comment:
This looks wrong; the constraint should be more general, and it should work if the value is wider than 64 bits.